### PR TITLE
syncd-rpc.mk: Fix stretch dockers build failure

### PR DESCRIPTION
**Pull request by renukamanavalan**
**- What I did**
The base syncd dockers follow a template, which defines the base docker as DOCKER_SYNCD_BASE instead of DOCKER_SYNCD_<platform code>. Fix the docker-syncd-<mlnx, bfn>.mk to use the new one.

[Note: I am yet to test it locally]
**- How I did it**

**- How to verify it**
build syncd-rpc dockers to succeed.

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/mellanox/docker-saiserver-mlnx.mk [hunk 1]
```diff
@@ -11,4 +11,4 @@ $(DOCKER_SAISERVER_MLNX)_CONTAINER_NAME = saiserver
 $(DOCKER_SAISERVER_MLNX)_RUN_OPT += --net=host --privileged -t
 $(DOCKER_SAISERVER_MLNX)_RUN_OPT += -v /host/machine.conf:/etc/machine.conf
 $(DOCKER_SAISERVER_MLNX)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
-$(DOCKER_SYNCD_MLNX)_RUN_OPT += --tmpfs /run/criu
+$(DOCKER_SYNCD_BAsE)_RUN_OPT += --tmpfs /run/criu
```
Comment (lguohan) [inline platform/mellanox/docker-saiserver-mlnx.mk]: BAsE? typo?

File: platform/mellanox/docker-syncd-mlnx-rpc.mk [hunk 2]
```diff
@@ -11,7 +11,7 @@ $(DOCKER_SYNCD_MLNX_RPC)_DEPENDS += $(SYNCD_RPC_DBG) \
 endif
 
 $(DOCKER_SYNCD_MLNX_RPC)_PYTHON_DEBS += $(MLNX_SFPD)
-$(DOCKER_SYNCD_MLNX_RPC)_LOAD_DOCKERS += $(DOCKER_SYNCD_MLNX)
+$(DOCKER_SYNCD_MLNX_RPC)_LOAD_DOCKERS += $(DOCKER_SYNCD_BAsE)
```
Comment (lguohan) [inline platform/mellanox/docker-syncd-mlnx-rpc.mk]: typo?

File: platform/barefoot/docker-syncd-bfn-rpc.mk [hunk 3]
```diff
@@ -9,7 +9,7 @@ $(DOCKER_SYNCD_BFN_RPC)_DEPENDS += $(SYNCD_RPC_DBG) \
                                    $(LIBSAIMETADATA_DBG) \
                                    $(LIBSAIREDIS_DBG)
 endif
-$(DOCKER_SYNCD_BFN_RPC)_LOAD_DOCKERS += $(DOCKER_SYNCD_BFN)
+$(DOCKER_SYNCD_BFN_RPC)_LOAD_DOCKERS += $(DOCKER_SYNCD_BASE)
```
Comment (lguohan) [inline platform/barefoot/docker-syncd-bfn-rpc.mk]: why it is base? rpc docker is based on syncd.

File: platform/mellanox/docker-syncd-mlnx-rpc.mk (hunk 2 above)
Comment (renukamanavalan) [reply platform/mellanox/docker-syncd-mlnx-rpc.mk]: Corrected. Thank you!

File: platform/barefoot/docker-syncd-bfn-rpc.mk (hunk 3 above)
Comment (renukamanavalan) [reply platform/barefoot/docker-syncd-bfn-rpc.mk]: DOCKER_SYNCD_BASE expands to docker-syncd-$(DOCKER_SYNCD_PLATFORM_CODE).gz,  which in this case would be docker-syncd-bfn.gz

**Comment by renukamanavalan** [comment]
retest this please

**Comment by renukamanavalan** [comment]
retest this please

**Pull request merged**